crates/lox_core/src/time/leap_seconds/api.rs

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2023. Helge Eichhorn, Andrei-Costin Zisu and the LOX
```

Comment:
@helgee  this code derives from your original MIT code in LeapSeconds.jl. Please confirm here that you approve of this relicensing.

Reply:
@matzipan @AngusGMorrison Should we adopt [REUSE](https://reuse.software/)? They provide a Python tool to handle headers and compliance checking.
There is also [All Contributors](https://allcontributors.org/) but I find it to noisy and having the contributor data only in an HTML table in a Markdown doc is ugly.

Reply:
I am not sure how REUSE works, I haven't come across it before.

Reply:
Changed the license header as requested.